submissions/ik-web/memory-pair-game/js/app.js

```diff
@@ -0,0 +1,133 @@
+document.addEventListener("DOMContentLoaded", () => {
+
+    const cards = [
+        {
+            id: 1,
+            src: 'img/barbossa.jpg'
+        },
+        {
+            id: 2,
+            src: 'img/jack.jpg'
+        },
+        {
+            id: 3,
+            src: 'img/davy.jpg'
+        },
+        {
+            id: 4,
+            src: 'img/will.jpg'
+        },
+        {
+            id: 5,
+            src: 'img/elizabeth.jpg'
+        },
+        {
+            id: 6,
+            src: 'img/teach.jpg'
+        }
+    ];
+
+    const gameName = document.querySelector('#title');
+    const playArea = document.querySelector('#play-area');
+    const playCardsTotal = cards.length * 2;
+
+    const checkPairDelay = 800;
+    const closeCardsDelay = 800;
+    const hideCardsDelay = 600;
+    const delayEndGame = 900;
+    
+    function ShuffleArr(arr) {
+        for(let i = 0; i < arr.length; i++) {
+            let k = Math.floor( Math.random() * arr.length);
+            [arr[i], arr[k]] = [arr[k], arr[i]];
+        }
+        return arr;
+    }
+    
+    function createCard(cardId, cardImageSrc) {
+        return `
+        <div class="card" data-id="${cardId}"> 
+            <div class="flipper"> 
+                <div class="front">
+                    <img src="${cardImageSrc}" class="image">
+                </div>              
+                <div class="back"></div>                
+            </div>        
+        </div>
+    `;
+    }
+    
+    function addCardsToPlayArea(cards) {
+        const playCards = cards.concat(cards);
+        let ShuffledCards = '';
+    
+        ShuffleArr(playCards).forEach((playCard) => ShuffledCards += createCard(playCard.id, playCard.src));
+        playArea.innerHTML = ShuffledCards;
+    }
+    
+    function clickCard() {
+        playArea.addEventListener('click', checkCard);
+    }
+
+    function checkCard({target}) {
+        let clickTarget = target.closest('div.card');
+        if (!clickTarget) return;
+        clickTarget.classList.add('open');
+
+        checkPair();
+    }
+
+    function checkPair() {
+        const openedCards = [...document.querySelectorAll('.open')];
+        
+        if (openedCards.length === 2) {
+            stopClick();
+            
+            if (openedCards[0].getAttribute('data-id') === openedCards[1].getAttribute('data-id')) {
```

Comment:
use destructuring, it will help you have more readable code
`const [firstCard, secondCard] = openedCards;`

Reply:
Fixed